geo_concerns/app/models/image.rb

```diff
@@ -0,0 +1,7 @@
+# Base class for Images that are georeferenced with a bbox
+class Image < ActiveFedora::Base
+  include ::GeoConcerns::ImageBehavior
+  validates_presence_of :title,  message: 'Your work must have a title.'
+  validates_presence_of :georss_box,  message: 'Your work must have a bbox.'
```

Comment:
Good call. We should determine what properties should be enforced.

1. title
2. bbox
3. ??